Add `Lazer Grand Arena: Registrations now open!` wiki article and news post (#11599)
ppy/osu-wiki: wiki/Tournaments/LGA/2024/en.md

---
tags:
  - lazer
  - grand
  - arena
  - osulazer
  - grand arena
  - lazer grand arena
---

# Lazer Grand Arena

![LGA 2024 banner](img/lga2024-banner.jpg)

The **Lazer Grand Arena** (***LGA***) is an osu!(lazer) tournament hosted by the [osu! team](/wiki/People/osu!_team).

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-06-18 (10:00 UTC)/2024-07-01 (23:59 UTC) |
| Qualifier and tournament format showcase | 2024-07-07 (14:00 UTC) |
| Qualifier stage | 2024-07-13/2024-07-14 |
| Bracket stage (single elimination) | 2024-07-20/2024-07-21 |

> | Top 8 (single elimination)/Top 4 (double elimination) | 2024-07-27/2024-07-28 |

Reviewer: sentence case

```suggestion
| Top 8 (single elimination)/top 4 (double elimination) | 2024-07-27/2024-07-28 |
```

## Prizes

The minimum prize pool will be revealed after the Qualifier stage.

| Placing | Prizes |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | profile badge, %50 of the prize pool, *more prizes TBA* |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, %50 of the prize pool, *more prizes TBA* |
```

    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches and must be exchanged for the duration of the punishment.
13. Penalties for violating the tournament rules include, but are not limited to:
    - Removal of pick rights for a game or match.
    - Point deductions.
    - Declaring the match as forfeited, or as a win by default for the other player.
    - Disqualification from the entire tournament.
    - Disqualification from the current and future official tournaments, until appealed.
14. Referees may allow, at their discretion, lower or higher tolerances for timers.
15. The tournament managers may request liveplays or recordings of individual players at any point in the tournament without prior warning.
16. The tournament managers will be responsible for receiving and investigating any tournament-related complaints.
17. The tournament managers reserve the right to modify these rules at any moment. Any such changes will be announced in advance.

### Tournament registration

1. Every user interested in will be required to signs up by utilising the osu! tournament listing [linked above](#links).

> 2. To ensure valid registrations, every prospective participant will be manually checked by the [account support team](/wiki/People/Account_support_team), in a manner similar to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening) that is offered to community tournaments. To be successfully accepted on the list, players are required to be placed #2500 or higher on the osu! global ranking, and not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.

Reviewer: ```suggestion
2. To ensure valid registrations, every prospective participant will be manually checked by the [account support team](/wiki/People/Account_support_team), in a manner similar to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening) that is offered to community tournaments. To be successfully accepted on the list, players are required to be placed #2,500 or higher on the osu! global ranking, and not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
```

5. Tournament staff members are **not** allowed to play in the tournament, with the exception of commentators, statisticians, and streamers.
   - The tournament managers must ensure that such staff members do not have access to priviledged information, e.g. Qualifiers score information, mappool information before the mappool showcase, etc. They are to be treated as regular players whenever possible.

### Qualifier information

1. In the Qualifier stage, all players will play a specific pool designed by the mappool selectors.
2. The mappool will contain 4 brackets: No Mod, [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), and [Double Time](/wiki/Gameplay/Game_modifier/Double_Time).
3. The Qualifier pool will feature 11 beatmaps, organised as such:
   - No Mod: 4 beatmaps
   - Hidden: 2 beatmaps
   - Hard Rock: 2 beatmaps
   - Double Time: 3 beatmaps
4. Players will be asked to play the mappool twice at a designated time. The best play for each map will be used for seeding.
5. The mappool is to be played according to the order listed on this page.

> 6. Only the top 32 seeded players will advance to the Bracket stage.

Reviewer: not capitalised elsewhere on this page

```suggestion
6. Only the top 32 seeded players will advance to the bracket stage.
```

3. Following the Qualifiers, the bracket stage will have two phases (one per weekend):
   - For the first weekend, a single-elimination bracket will be played from the Round of 32 up until the Quarterfinals.
     - All matches in this phase will utilise the same mappool. The first player to score 3 points wins (best of 5). 
     - The first weekend's pool will feature 11 maps, organised as such:
     - No Mod: 4 beatmaps
     - Hidden: 2 beatmaps
     - Hard Rock: 2 beatmaps
     - Double Time: 3 beatmaps
    - The draft order will be as follows:
      1. The first player (A) will ban one beatmap, followed by the second player (B) also banning a beatmap: AB
      2. Players will pick two beatmaps respecting the following order: BAAB
      3. Both players ban will two maps, as such: ABBA
      4. The last beatmap remaining in the pool will be used as the 5th pick for the match.
   - For the second weekend, The 8 remaining players from the previous phase will then then play another set of matches, after which the remaining 4 players will be placed on a double-elimination bracket.

>      - All matches in this phase will utilise the same mappool, except for the Losers bracket Finals and Grand Finals, where a new osu! original will be introduced into the pool. The first player to score 4 points wins (best of 7).

Reviewer: probably like so? i've generally seen two more common ways of writing it and i'm pretty sure both include the apostrophe. going by https://en.wikipedia.org/wiki/Double-elimination_tournament

```suggestion
     - All matches in this phase will utilise the same mappool, except for the Losers' Bracket Finals and Grand Finals, where a new [osu! original](/wiki/osu!_originals) will be introduced into the pool. The first player to score 4 points wins (best of 7).
```

     - The second weekend's pool will feature 15 maps, organised as such:
       - No Mod: 5 beatmaps
       - Hidden: 3 beatmaps
       - Hard Rock: 3 beatmaps
       - Double Time: 4 beatmaps
     - The draft order will be as follows:
       1. The first player (A) will ban one beatmap, followed by the second player (B) also banning a beatmap: AB
       2. Players will pick two beatmaps respecting the following order: BAAB
       3. Both players will ban two beatmaps, as such: ABBA
       4. Both players will pick one beatmap: AB
       5. Both players will ban one beatmap: BA

>        6. The last beatmap remaining in the pool will be used as the 7th pick for the match

Reviewer: ```suggestion
       6. The last beatmap remaining in the pool will be used as the 7th pick for the match.
```

>        7. Exceptionally, for the Losers bracket Finals and Grand Finals, steps 5 and 6 will not be applied, and the last pick will be an osu! original beatmap, to be released at match time.

Reviewer: ```suggestion
       7. Exceptionally, for the Losers' Bracket Finals and Grand Finals, steps 5 and 6 will not be applied, and the last pick will be an osu! original beatmap, to be released at match time.
```

### Win conditions

- In the Qualifiers, players need to place in the top 32 in order to advance to the single-elimination bracket stage.
  - Seeding for this stage will follow the %MAX format — the highest score for each map will receive 100% of the points (i.e. 1 point), and every other player will be awarded a percentage of that top score. The sum of the highest score for every map will be used as the final seeding, with a higher point sum meaning higher placement (i.e. sorted in descending order).
    - The exact formula that will be used for all players (and for each map) is `Final Score = Player Map Score / MAX Score`, where:
      - `Final Score` is the numerical value awarded to the current player for each map.

>       - `Player Map Score` is the score the player being evaluated achieved on the current map, the best score out of two is taken.

Reviewer: ```suggestion
      - `Player Map Score` is the score the player being evaluated achieved on the current map, taking the best score out of two.
```

      - `MAX Score` is the highest score achieved for the current map across all players and all runs.
- In the Round of 32 and Round of 16, players need to win 3 maps to win a match (best of 5).
- In the Quarterfinals, Semifinals, Finals, and Grand Finals, players need to win 4 maps to win a match (best of 7).

>   - *Note: for the Grand Finals match, the player that is coming from the winners bracket is required to win one "set", that is, one best of 7, to win the tournament. The player coming from the losers bracket, however, needs to win 2 sets.*

Reviewer: ```suggestion
  - *Note: for the Grand Finals match, the player that is coming from the winners' bracket is required to win one "set", that is, one best of 7, to win the tournament. The player coming from the losers' bracket, however, needs to win 2 sets.*
```

### Match procedures

> 1. A referee will create a multiplayer room 10 minutes before the scheduled match time. players must join the lobby in that period.

Reviewer: ```suggestion
1. A referee will create a multiplayer room 10 minutes before the scheduled match time. Players must join the lobby in that period.
```

3. Players will be allowed 2 minutes for each pick/ban phase (as described on the [bracket stage instructions](#bracket-stage-information) section), and an additional 2 minutes to press the `Ready` button on their client before play commences. If a player takes more time than allotted for either action, the procedures adopted will be as follows:
   - For the first occurrence:
     - The player will receive a verbal warning from the referee; in addition, the referee may opt to forcefully use the player's tactical timeout to allow for the player to pick.
     - If the player does not pick a map within 15 seconds from when the timer expires, the referee will apply the procedure for subsequent occurrences listed below, i.e. a random pick will be used for a pick timer, and the match will be forcefully started for a ready timer.
   - On subsequent occurrences:
     - For a pick timer: a random map will be chosen from the mappool by rolling a random number, based on the amount of beatmaps that were neither picked nor banned.

>      - For a ready timer: the referee will issue the start command on a 10 second countdown. In this scenario, if the referee starts the match and either player is not present in the lobby, their opponent will receive the point instead.

Reviewer: ```suggestion
     - For a ready timer: the referee will issue the start command on a 10-second countdown. In this scenario, if the referee starts the match and either player is not present in the lobby, their opponent will receive the point instead.
```

     - Repeat offenders may receive further sanctions from the tournament managers.

### Scheduling information

1. Each stage (as described on the [Tournament schedule](#tournament-schedule) table) will be held on **a single weekend**.
2. Matches in the Qualifiers will be held during specific time slots proposed by the tournament managers. Each player may choose a time slot in which they want to play.

>    - **Players that do not pick a time slot by Thursday, 12 July 2024 (23:59 UTC) will be disqualified from the competition.**

Reviewer: ```suggestion
   - **Players who do not pick a time slot by Thursday, 12 July 2024 (23:59 UTC) will be disqualified from the competition.**
```
